httpobs/website/api_v2.py

```diff
@@ -0,0 +1,134 @@
+import sys
+from datetime import datetime, timedelta
+
+from flask import Blueprint, jsonify, request
+
+import httpobs.database as database
+import httpobs.scanner as scanner
+from httpobs import STATE_FAILED
+from httpobs.conf import API_COOLDOWN, DEVELOPMENT_MODE
+from httpobs.scanner.grader import get_score_description
+from httpobs.website import add_response_headers
+from httpobs.website.utils import valid_hostname
+
+api_v2 = Blueprint("api_v2", __name__)
+
+
+@api_v2.route("/analyze", methods=["GET", "OPTIONS", "POST"])
```

Comment:
How does this handle OPTIONS requests?

Reply:
Through the `@add_response_headers(cors=True)` decorator